src/components/Admin/SystemLogs/SystemLogs.js

```diff
@@ -8,15 +8,31 @@ import urls from '../../../utils/urls';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import Alert from '../../shared/Alert';
 import styled from 'styled-components';
+import isMobileView from '../../../utils/isMobileView';
+
+const mobileView = isMobileView();
```

Comment:
MobileView not required here, use media queries

Reply:
@fragm3 I passing left as a prop to shared Alert component. So I guess I cannot use meadia queries

Reply:
@fragm3 The screenshot you have attached will occur if you do not reload in mobile view. There is no UI problem if we reload it in mobile view. Please check

Reply:
I don't understand the point of using props for it and passing it down to alert component. You could have directly used media queries in alert component.